### PR TITLE
Hosting: Answer common questions for SFTP/DB inline to the card, also add missing analytics

### PR DESCRIPTION
**Pull request by gwwar**
fixes https://github.com/Automattic/wp-calypso/issues/37890

This PR updates SFTP/DB to answer top questions, also updates the subheader to "Access and manage more advanced settings of your website." Most copy was taken from @automattic-ian's new support docs 💖.  

I've also updated the SFTP card to have a copy button for each field, but I'm sure we may want to style this a bit more:

Screen Shot 2019-11-22 at 5 19 07 PM

accordian

| Original  | SFTP Not Enabled | SFTP Enabled | Small Viewport |
| ------------- | ------------- |------------- |------------- |
| Screen Shot 2019-11-22 at 2 44 56 PM | Screen Shot 2019-11-22 at 2 39 18 PM |  Screen Shot 2019-11-22 at 2 45 49 PM | Screen Shot 2019-11-22 at 2 50 05 PM|

I've also opened a related issue https://github.com/Automattic/wp-calypso/issues/37899

### Testing Instructions
- Create a new site, with a business plan. Transfer to atomic by installing a plugin.
- Visit http://calypso.localhost:3000/hosting-config/
- Verify that styling copy make sense
- Verify that we can still enable SFTP credentials and DB access

#### Analytics
- In console set `localStorage.debug='calypso:analytics*';`
- Clicking on "Contact Us" triggers the following analytics events:
Screen Shot 2019-11-22 at 3 36 03 PM
- Clicking on "Reset password" in the SFTP card triggers the following analytics events
Screen Shot 2019-11-22 at 3 47 19 PM
- Clicking on "Open phpMyAdmin" in the Database access card triggers the following events:
Screen Shot 2019-11-22 at 4 02 14 PM
- Clicking on "Restore" in the dialog of the Database Access card triggers the following events
Screen Shot 2019-11-22 at 4 12 18 PM
- Clicking on Enable SFTP Credentials triggers the following events:
Screen Shot 2019-11-22 at 4 17 14 PM

### CODE REVIEW

File: client/my-sites/hosting/phpmyadmin-card/index.js [hunk 1]
```diff
@@ -61,9 +62,21 @@ const PhpMyAdminCard = ( { translate, siteId, token, loading, disabled } ) => {
 			<CardHeading>{ translate( 'Database Access' ) }</CardHeading>
 			<p>
 				{ translate(
-					'Manage your database with phpMyAdmin and run a wide range of operations with MySQL.'
+					'For the tech-savvy, manage your database with phpMyAdmin and run a wide range of operations with MySQL.'
```
Comment (a8ci18n) [inline client/my-sites/hosting/phpmyadmin-card/index.js]: :warning: This change will be queued for retranslation. We'll display the English text until then.
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'Manage your database with phpMyAdmin and run a wide range of operations with MySQL.' )` *ES Score: 6*

File: client/my-sites/hosting/sftp-card/index.js [hunk 2]
```diff
@@ -116,28 +117,51 @@ const SftpCard = ( {
...
+
 	return (
 		<Card className="sftp-card">
 			<MaterialIcon icon="cloud" size={ 32 } />
-			<CardHeading>{ translate( 'SFTP Information' ) }</CardHeading>
+			<CardHeading>{ translate( 'SFTP Credentials' ) }</CardHeading>
 			<div className="sftp-card__body">
-				{ disabled || username || isLoading ? (
+				<p>
+					{ translate( "Access and edit your website's files directly by using an FTP client." ) }
```
Comment (a8ci18n) [inline client/my-sites/hosting/sftp-card/index.js]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'Access and edit your website's files directly using an FTP client.' )` *ES Score: 8*

**Comment by gwwar** [comment]
@Automattic/serenity @olaolusoga Folks are welcome to commit directly to the branch/take this over, though we probably shouldn't bunch any more changes than this, before landing this piece.

File: client/my-sites/hosting/sftp-card/index.js [hunk 3]
```diff
@@ -192,8 +278,8 @@ export default connect(
 
 		if ( ! disabled ) {
 			const sftpDetails = getAtomicHostingSftpUser( state, siteId, currentUserId );
-			username = get( sftpDetails, 'username' );
-			password = get( sftpDetails, 'password' );
+			username = sftpDetails?.username;
```
Comment (glendaviesnz) [inline client/my-sites/hosting/sftp-card/index.js]: 🎉

File: client/my-sites/hosting/sftp-card/index.js [hunk 4]
```diff
@@ -116,41 +124,96 @@ const SftpCard = ( {
...
-						<Button onClick={ createUser } primary>
-							{ translate( 'Enable SFTP' ) }
-						</Button>
-					</>
-				) }
+				<p>
+					{ translate( "Access and edit your website's files directly by using an SFTP client." ) }
+				</p>
 			</div>
+			{ displayQuestionsAndButton && (
```
Comment (glendaviesnz) [inline client/my-sites/hosting/sftp-card/index.js]: Starting to get to the point where some child components for the enabled and non-enabled states might simplify this component, but we can look at that when we circle back to allow multiple ftp users.

File: client/my-sites/hosting/sftp-card/index.js [hunk 5]
```diff
@@ -116,41 +124,96 @@ const SftpCard = ( {
...
+								</>
+							</td>
+						</tr>
+						<tr key={ translate( 'Port' ) }>
+							<th>{ translate( 'Port' ) }:</th>
+							<td>
+								<>
+									<span>{ ! disabled && SFTP_PORT }</span>
+									<ClipboardButton
+										text={ SFTP_PORT.toString() }
```
Comment (glendaviesnz) [inline client/my-sites/hosting/sftp-card/index.js]: ClipboardButton didn't like a number being passed into the text prop - worked, but threw a console error - I fixed this.

**Comment by glendaviesnz** [comment]
Decided to land as it will be easier to do #37580 with these changes in place - @olaolusoga, feel free to open a new issue with any outstanding design issues related to these changes.

**Comment by carolynannewells** [comment]
@glendaviesnz Hi Glen, jumping into this thread from editorial to help you with copy review. Please, can you clarify what you would like reviewed? Thanks so much!

**Comment by carolynannewells** [comment]
Hi @gwwar — I am just jumping into this thread from editorial to help with copy reviews. Please, can you clarify what you would like reviewed? Thank you so much!

**Pull request merged**